partcad/src/partcad/shape.py

```
@@ -562,326 +561,322 @@ def error(self, msg: str):
+
+    async def render_svg_somewhere(
+        self,
+        ctx,
+        project=None,
+        filepath=None,
+        line_weight=None,
+        viewport_origin=None,
+    ):
+        """Renders an SVG file somewhere, ignoring where the project wants it."""
+        if filepath is None:
+            filepath = tempfile.mktemp(".svg")
```

Ruff (0.16.1)

[error] Use of insecure and deprecated function (`mktemp`)

(S306)